invoice.go

```diff
@@ -88,6 +130,31 @@ type InvoiceDiscountParams struct {
 	Discount *string `form:"discount"`
 }
 
+// InvoicePaymentSettingsPaymentMethodOptionsBancontactParams is the set of parameters allowed for
+// bancontact on payment_method_options on payment_settings on an invoice.
+type InvoicePaymentSettingsPaymentMethodOptionsBancontactParams struct {
+	PreferredLanguage *string `form:"preferred_language"`
+}
+
+// InvoicePaymentSettingsParams is the set of parameters allowed for card on payment_method_options on
```

Comment:
Comment references the wrong type

Reply:
This is a bit nitpicky too, but I'd probably leave out things like `payment_method_options` just because the user will never directly interact with those API parameter names (i.e. they use Go symbols like `PaymentMethodOptions` instead).

I usually just keep the docs on these structs as simple as plausibly possible like "is the set of parameters for invoice payment settings". Not super useful I know, but pretty much no documentation will be unless we go for all-the-way API ref style docs (which might be possible with codegen).